ev3dev2/motor.py

```diff
@@ -1022,6 +1022,9 @@ class MediumMotor(Motor):
 
     def __init__(self, address=None, name_pattern=SYSTEM_DEVICE_NAME_CONVENTION, name_exact=False, **kwargs):
 
+        if platform in ('brickpi', 'brickpi3'):
```

Comment:
This PR has been superseded my other changes; should I port this forward though?